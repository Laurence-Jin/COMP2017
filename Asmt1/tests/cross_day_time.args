Wirragulla Cardiff "23:40:00"
