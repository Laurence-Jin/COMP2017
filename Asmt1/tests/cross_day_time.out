The next train to Cardiff from Wirragulla departs at 00:08:00
